src/utils/cid.ts

```diff
@@ -29,7 +29,7 @@ const codecs = {
  * @throws {Error} encoding fails
  * @throws {Error} if hasher is not supported
  */
-export async function computeCid(payload: any, codecCode = cbor.code, multihashCode = sha256.code): Promise<string> {
+export async function computeCid(payload: any, codecCode: number = cbor.code, multihashCode: number = sha256.code): Promise<string> {
```

Comment:
Nit: this line is on the long side. do you mind linebreaking it somewhere?

Reply:
Done. Note that we do have a linter rule on max 150 chars.

Reply:
ah didn't realize. i'm used to closer to 80-100 max chars

Reply:
80 to 100?!?! That was the standard length limit back when I was in college in 2000 when the screen size was 15" and resolution was 1024x768..... we now have 32"+ monitors with crazy resolutions!! = ]